docroot/themes/custom/mass_theme/mass_theme.theme

```diff
@@ -7235,10 +7235,7 @@ function mass_theme_preprocess_paragraph__social_media(&$variables) {
 function mass_theme_preprocess_paragraph__organization_contact_logo(&$variables) {
   $paragraph = $variables['paragraph'];
   $node = Helper::getParentNode($paragraph);
-
-  $variables['org_contact'] = $node->get('field_ref_contact_info_1')->view('contact_us');
-  $variables['org_logo'] = $node->get('field_sub_brand')->view('logo_link');
```

Comment:
@tom-fleming, we still need to get the the org logo - can you please get the logo? I can pass it in, or here's the new data structure  https://github.com/massgov/mayflower/blob/38c3e936085f6884045f8ab4c036d05c8c024fe6/packages/patternlab/styleguide/source/_patterns/03-organisms/contact/contact-row.json#L10-L15

After:
https://pr2277-ldza8sb4bnrzvx9sawozuxsnnyxmypgp.tugboatqa.com/orgs/governor-maura-healey-and-lt-governor-kim-driscoll
Before:
https://www.mass.gov/orgs/governor-maura-healey-and-lt-governor-kim-driscoll